### PR TITLE
Fix some documentation issues

### PR DESCRIPTION
**Pull request by ueman**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [x] Documentation

## :scroll: Description

I've fixed some documentation issues:
- https://pub.dev/documentation/sentry_flutter/latest/
- https://pub.dev/documentation/sentry_flutter/latest/sentry_flutter/SentryFlutterOptions-class.html
- https://pub.dev/documentation/sentry_flutter/latest/sentry_flutter/SentryNavigatorObserver-class.html

## :bulb: Motivation and Context

This is kinda obvious.

## :green_heart: How did you test it?
No code was changed, just documentation.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
None

### CODE REVIEW

File: flutter/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ In order to track navigation events you have to add the
 `SentryNavigationObserver` to your `MaterialApp`, `WidgetsApp` or `CupertinoApp`.
 
 ```dart
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
```
Comment (ueman) [inline flutter/README.md:64]: For `MaterialApp` one needs to import `import 'package:flutter/material.dart';`.

Comment (marandaneto) [reply flutter/README.md:64]: good catch

**Comment by marandaneto** [comment]
thanks @ueman :)

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**